pt-BR/upgrading_ruby_on_rails.md

```diff
@@ -1,33 +1,36 @@
 **DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
 
-Upgrading Ruby on Rails
+Atualizando Ruby on Rails
 =======================
 
-This guide provides steps to be followed when you upgrade your applications to a newer version of Ruby on Rails. These steps are also available in individual release guides.
+Este guia fornece os passos a serem seguidos quando você for atualizar suas aplicações
+para uma versão mais nova do Ruby on Rails. Estes passos também estão disponíveis em guias de releases individuais.
 
 --------------------------------------------------------------------------------
 
-General Advice
+Conselho Geral
 --------------
 
-Before attempting to upgrade an existing application, you should be sure you have a good reason to upgrade. You need to balance several factors: the need for new features, the increasing difficulty of finding support for old code, and your available time and skills, to name a few.
+Antes de tentar atualizar uma aplicação existente, você deve ter certeza que possui uma boa razão para faze-lo. Então tenha em mente alguns fatores: a necessidade de novas funcionalidades, a crescente dificuldade de encontrar suporte para código mais antigo, seu tempo disponível e habilidades, por exemplo
```

Comment:
Acho que aqui faltou o ponto final. Estou pensando em algo para o `to name a few.` talvez `para citar alguns` ou `entre outros`? O que vocês acham? @erikacamposdesign @joaorsalmeida @Marcelo-Zanini

Reply:
`entre outros` acredito que resolve bem sim.